tests/test_django.py

```diff
@@ -95,8 +97,10 @@ def test_serialization(self):
         data = serializers.serialize('json', [o1, o2])
 
         obj1, obj2 = serializers.deserialize('json', data)
-        self.assertEqual(o1, obj1.object)
-        self.assertEqual(o2, obj2.object)
+        self.assertEqual(o1.version, obj1.object.version)
```

Comment:
Here and below, these assertions only worked before due to a bug in django. It used to compare models based on equality of their ``pk`` attributes. In this case, both were None, which were considered equal. Starting in django 1.7 that logic was improved, and these tests no longer passed.